src/components/Achievements.jsx

```diff
@@ -92,75 +96,37 @@ const Achievements = () => {
           {props.event}
         </p>
         <p className="font-poppins italic font-normal text-lg text-gradient mb-3">
           {props.position}
         </p>
-        {props.content1 && (
-          <p className="font-poppins font-normal text-dimWhite text-sm mb-1">
-            🚀 {props.content1}
-          </p>
-        )}
-        {props.content2 && (
-          <p className="font-poppins font-normal text-dimWhite text-sm mb-1">
-            ⚡ {props.content2}
-          </p>
-        )}
-        {props.content3 && (
-          <p className="font-poppins font-normal text-dimWhite text-sm mb-4">
-            🔥 {props.content3}
-          </p>
-        )}
+        {props.content1 && <p className="font-poppins font-normal text-dimWhite text-sm mb-1">🚀 {props.content1}</p>}
+        {props.content2 && <p className="font-poppins font-normal text-dimWhite text-sm mb-1">⚡ {props.content2}</p>}
+        {props.content3 && <p className="font-poppins font-normal text-dimWhite text-sm mb-4">🔥 {props.content3}</p>}
       </div>
-
       <div className="flex flex-row mb-2 font-poppins font-normal text-dimWhite">
         {props.article && (
-          <a
-            className="inline-flex items-center mr-2 hover:text-teal-200"
-            href={props.article}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
+          <a className="inline-flex items-center mr-2 hover:text-teal-200" href={props.article} target="_blank" rel="noopener noreferrer">
             <TiNews size="1.5rem" className="inline" />
           </a>
         )}
         {props.youtube && (
-          <a
-            className="inline-flex items-center mr-2 hover:text-teal-200"
-            href={props.youtube}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
+          <a className="inline-flex items-center mr-2 hover:text-teal-200" href={props.youtube} target="_blank" rel="noopener noreferrer">
             <FaYoutube size="1.5rem" className="inline" />
           </a>
         )}
         {props.github && (
-          <a
-            className="inline-flex items-center mr-2 hover:text-teal-200"
-            href={props.github}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
+          <a className="inline-flex items-center mr-2 hover:text-teal-200" href={props.github} target="_blank" rel="noopener noreferrer">
             <AiFillGithub size="1.5rem" className="inline" />
           </a>
         )}
         {props.project && (
-          <a
-            className=" inline-flex items-center hover:text-teal-200"
-            href={props.project}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
+          <a className=" inline-flex items-center hover:text-teal-200" href={props.project} target="_blank" rel="noopener noreferrer">
             <BsLink45Deg size="1.5rem" className="inline" />
           </a>
         )}
```

Comment:
Can you revert the linter here?

Reply:
Well, you dont have to revert the commit. You can always make a new commit for the changes.

And making a new branch is not required, rather discouraged because often there are multiple rounds of reviews on the same PR. We can't then keep making new branches right? :) 
So its better to keep the discussion in one place.

And I squash the commits before merging, so commits will be clean. However, this might depend on the project. Regardless, having multiple commits is never an issue.

Reply:
got it. Thanks for explaining it to me :)